### DISCUSSION TITLE
Does readyset works with RDS proxy?

### DISCUSSION THREAD

**Discussion by schosterbarak** [readysettech/readyset#18] [Q&A] [answered]
We have an environment of AWS lambdas, RDS proxy and RDS aurora (postgres). 
What would be the best way to deploy readyset in that case?

**Comment by jbensal** [comment, marked as answer]
Hi @schosterbarak,

Thanks for your question! We'd suggest deploying ReadySet alongside your RDS proxy for now. Then you can have your lambdas send your queries to ReadySet.

We haven't fully dove in to the limitations with ReadySet + RDS Proxy yet. Do you have time to chat on a call to talk through this further?

Hope this helps.
Jay
